extra/3-magic-8-ball.js

```diff
@@ -60,8 +67,25 @@ function shakeBall() {
 */
 function checkAnswer(answer) {
   //Write your code in here
+  let answerNumber;
+  for(let i =0; i<20; i++){
+    if(answer === answers[i]){
+        answerNumber = i;
+    }
+  }
+  if(answerNumber < 5){
+    return "very positive";
+  } else if(answerNumber >=5 && answerNumber < 10){
+    return "positive";
+  } else if(answerNumber >=10 && answerNumber < 15){
```

Comment:
Nice Musa, really clean solution!

Reply:
There is a slightly even cleaner way of writing the if and else blocks here using something called ternary operators, if you want to add another stretch goal have a look into these and have a think about how that would look here

Reply:
Thank you for your comment... actually, I was thinking to ask about How can I improve that solution... I will look at the " ternary operators" Thank you

Reply:
Added Ternary operators ...